src/test/java/edu/kimjones/advancedjava/stock/StockQuoteApplicationTest.java

```diff
@@ -18,12 +20,21 @@ public void testMainPositive() {
         args[1] = "9/20/2018";
         args[2] = "9/23/2018";
 
-        StockQuoteApplication.main(args);
+        try {
+            StockQuoteApplication.main(args);
+        }
+        catch(IOException e) {
+        }
     }
 
     @Test(expected = NullPointerException.class)
     public void testMainNegative() {
-        StockQuoteApplication.main(null);
+
+        try {
+            StockQuoteApplication.main(null);
+        }
+        catch(IOException e) {
```

Comment:
with     @Test(expected = NullPointerException.class)
you don't need the try / catch.
instead just add IOException to the method signature. 
You pretty much never want to have to write code that has an empty catch block.